.github/workflows/cws-btfhub-sync.yml

```diff
@@ -26,7 +26,7 @@ jobs:
       - name: Install python
         uses: actions/setup-python@v4
         with:
-          python-version: '3.9'
+          python-version-file: '.python3-version'
```

Comment:
Here is an example run for this job: https://github.com/DataDog/datadog-agent/actions/runs/6742552121/job/18328848192
TLDR there is no 3.9.5 for ubuntu latest (22.04).

Reply:
My bad ! Just found the available versions for [ubuntu 22.04](https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md#python) and [ubuntu 20.04](https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2004-Readme.md#python).

Bumping it to 3.9.18 !